Description: SKIP tests on m32 which use struct shm_info or struct msginfo
 These tests fail on amd64 on the buildds, but not locally.
 They were disabled in 0005-XFAIL-tests-on-m-32-which-use-struct-shm_info-or-s.patch
 in strace/5.9-0ubuntu2, but the patch was dropped from Debian when it no longer applied. 
Author: Graham Inggs <ginggs@debian.org>
Last-Update: 2021-08-26

--- a/tests-m32/ipc_msg-Xabbrev.gen.test
+++ b/tests-m32/ipc_msg-Xabbrev.gen.test
@@ -1,4 +1,5 @@
 #!/bin/sh -efu
 # Generated by ./tests/gen_tests.sh from ./tests/gen_tests.in (ipc_msg-Xabbrev +ipc.sh -Xabbrev -a26); do not edit.
+exit 77
 set -- -Xabbrev -a26
 . "${srcdir=.}/ipc.sh"
--- a/tests-m32/ipc_msg.gen.test
+++ b/tests-m32/ipc_msg.gen.test
@@ -1,4 +1,5 @@
 #!/bin/sh -efu
 # Generated by ./tests/gen_tests.sh from ./tests/gen_tests.in (ipc_msg +ipc.sh -a26); do not edit.
+exit 77
 set -- -a26
 . "${srcdir=.}/ipc.sh"
--- a/tests-m32/ipc_msg-Xraw.gen.test
+++ b/tests-m32/ipc_msg-Xraw.gen.test
@@ -1,4 +1,5 @@
 #!/bin/sh -efu
 # Generated by ./tests/gen_tests.sh from ./tests/gen_tests.in (ipc_msg-Xraw +ipc.sh -Xraw -a16); do not edit.
+exit 77
 set -- -Xraw -a16
 . "${srcdir=.}/ipc.sh"
--- a/tests-m32/ipc_msg-Xverbose.gen.test
+++ b/tests-m32/ipc_msg-Xverbose.gen.test
@@ -1,4 +1,5 @@
 #!/bin/sh -efu
 # Generated by ./tests/gen_tests.sh from ./tests/gen_tests.in (ipc_msg-Xverbose +ipc.sh -Xverbose -a34); do not edit.
+exit 77
 set -- -Xverbose -a34
 . "${srcdir=.}/ipc.sh"
--- a/tests-m32/ipc_shm.gen.test
+++ b/tests-m32/ipc_shm.gen.test
@@ -1,4 +1,5 @@
 #!/bin/sh -efu
 # Generated by ./tests/gen_tests.sh from ./tests/gen_tests.in (ipc_shm +ipc.sh -a26); do not edit.
+exit 77
 set -- -a26
 . "${srcdir=.}/ipc.sh"
--- a/tests-m32/ipc_shm-Xabbrev.gen.test
+++ b/tests-m32/ipc_shm-Xabbrev.gen.test
@@ -1,4 +1,5 @@
 #!/bin/sh -efu
 # Generated by ./tests/gen_tests.sh from ./tests/gen_tests.in (ipc_shm-Xabbrev +ipc.sh -Xabbrev -a26); do not edit.
+exit 77
 set -- -Xabbrev -a26
 . "${srcdir=.}/ipc.sh"
--- a/tests-m32/ipc_shm-Xraw.gen.test
+++ b/tests-m32/ipc_shm-Xraw.gen.test
@@ -1,4 +1,5 @@
 #!/bin/sh -efu
 # Generated by ./tests/gen_tests.sh from ./tests/gen_tests.in (ipc_shm-Xraw +ipc.sh -Xraw -a19); do not edit.
+exit 77
 set -- -Xraw -a19
 . "${srcdir=.}/ipc.sh"
--- a/tests-m32/ipc_shm-Xverbose.gen.test
+++ b/tests-m32/ipc_shm-Xverbose.gen.test
@@ -1,4 +1,5 @@
 #!/bin/sh -efu
 # Generated by ./tests/gen_tests.sh from ./tests/gen_tests.in (ipc_shm-Xverbose +ipc.sh -Xverbose -a34); do not edit.
+exit 77
 set -- -Xverbose -a34
 . "${srcdir=.}/ipc.sh"
